util.c

```diff
@@ -1932,6 +1932,58 @@ int mdmon_running(const char *devnm)
 	return 0;
 }
 
+/* wait_for_mdmon_control_socket() - Waits for mdmon control socket
```

Comment:
Please use kernel code style, generally it is:
```
/**
 * wait_for_mdmon_control_socket() - Waits for mdmon control socket.
 * @devnm: Device for which mdmon control socket should start.
 * 
 * In foreground mode, when mdadm is trying to connect to control
 * socket it is possible that the mdmon has not created it yet.
 *  Give some time to mdmon to create socket. Timeout set to 10 sec.
 */
```
Please read:
https://www.kernel.org/doc/html/v4.10/process/coding-style.html
for further reference.